configs/neox_arguments.md

```diff
@@ -111,7 +111,7 @@ Logging Arguments
 
 - **git_hash**: str
 
-    Default = 2da1083
+    Default = efaee8d
```

Comment:
these are just showing up because we didn't run the `Update NeoXArgs docs automatically` auto-commit in https://github.com/EleutherAI/gpt-neox/pull/1080

I'll just include them here.